drivers/clock_control/Kconfig.nrf

```diff
@@ -4,6 +4,15 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
+config CLOCK_CONTROL_NRF_FORCE_ALT
+	bool
+	depends on SOC_COMPATIBLE_NRF
+	help
+	  This option can be enabled by an out-of-tree library if it
```

Comment:
Can you  update the descriptions so that they reflect "force alternative implementation"?

Reply:
i.e. not refer to out-of-tree but rather keep it focused on what it actually does

Reply:
Changed.